### PR TITLE
add amy's club meeting 10-27

### PR DESCRIPTION
**Pull request by hellyeah**

### CODE REVIEW

File: case_studies/elizabeth_high_school/2015-10-27_meeting/Elizabeth High School - Amy Sorto - 10:27 [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+Main issues are the school wifi, blocking things such as making github accounts or stack overflow.
+
+They wanted the cs club's help to make a website for bilingual students in our city.
+
+Runs a newsletter for her club.
+
+The club is mainly sophomore/juniors now so we are set.
+
+"Last meeting we had elections for some positions, and then did some live coding on making a website. Also, our attendance went up from the last meeting! We have 25 members now!"
```
Comment (MaxWofford) [inline case_studies/elizabeth_high_school/2015-10-27_meeting/Elizabeth High School - Amy Sorto - 10:27]: Make sure  to character wrap this!

**Comment by jonleung** [comment]
@MaxWofford in the future if you could move any pull requests you see into the "Review" column, that would be great!

**Comment by MaxWofford** [comment]
@jonleung Aren't there scenarios where PRs should be "in progress"?

**Comment by MaxWofford** [comment]
@hackedu/core Thoughts on only putting PRs in review after they have been assigned to a reviewer?

**Comment by jonleung** [comment]
@MaxWofford good point!

**Comment by zachlatta** [comment]
Per the retrospectives last Friday (meeting notes [here](https://github.com/hackedu/meta/blob/master/meetings/15-10-30_retrospectives.md)), we should move the pull request to the `This Week` column after assigning a reviewer.

File: case_studies/elizabeth_high_school/2015-10-27_meeting/Elizabeth High School - Amy Sorto - 10:27 (hunk 1 above)
Comment (zachlatta) [inline case_studies/elizabeth_high_school/2015-10-27_meeting/Elizabeth High School - Amy Sorto - 10:27]: And to add a newline at the end of the file!

**Comment by hellyeah** [comment]
Updated with 80 character wrap.

**Comment by MaxWofford** [comment]
Closed by #476

**Pull request closed without merging**